ranged_weapon.cpp

```diff
@@ -39,6 +39,10 @@ RangedWeapon::RangedWeapon(AttrType attr, const string& name, ViewId id, int dis
 
 void RangedWeapon::fire(WCreature c, Vec2 dir) const {
   CHECK(dir.length8() == 1);
+  if (c->getAmmoHeld() < 1) {
+    c->getPosition().globalMessage("Searching for ammunition.");
```

Comment:
There's a bug here. 10 ammunition points = 1 arrow. But "Searching for ammunition" will use up any points between 0 and 10.